code/iLasso_for_SINGE.m

```diff
@@ -32,98 +32,111 @@
 numregs = length(pa);
 % Define function for Gaussian kernel
 gkern = @(x,y) gausskernel(x,y,SIG);
+% Check for number of branches, if greater than 1, load branches from the Temporary File
```

Comment:
There are lots of changes in this diff that only indicate moved code, not new code.  Let's discuss the parts that were added so I can focus on those more closely.